.github/workflows/python.yml

```diff
@@ -0,0 +1,38 @@
+name: Python package
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install codecov black
+        pip install .[dev]
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```

Comment:
Why is this `if` clause required?

Reply:
Ah, it's an artifact of the GA template. Don't know why I kept it, mesa doesn't have any requirements file. Which also explains why there is an if statement, but not why we need it. Should I delete the line?

Reply:
IMO just delete it. It will become confusing in the future without proper comment explaining why.

Reply:
You are probably right, I deleted it